### PR TITLE
feat(imageBlock, shared): add accessibility, extract set alt text flyout to shared and reuse in image block

### PR DESCRIPTION
**Pull request by celine-s**
[ticket](https://app.clickup.com/t/86782x7u0)

### CODE REVIEW

File: packages/shared/src/components/EditAltTextFlyout.tsx
Comment (ragi96) [inline packages/shared/src/components/EditAltTextFlyout.tsx:1]: Test are missing, and would love to have it in its own folder as the other shared components

File: packages/animation-curve-block/src/AnimationCurveBlock.spec.ct.tsx [hunk 1]
```diff
@@ -424,7 +424,6 @@ describe('AnimationCurve Block', () => {
         cy.get(FLYOUT_SELECTOR).should('exist');
```
[the remark this answers is not in the document]
Comment (celine-s) [reply packages/animation-curve-block/src/AnimationCurveBlock.spec.ct.tsx:424]: The test always failed. Imho it did not make sense; we move the end point to 0,50 butt afterwards test it and say it should equal 1 (line 431 +). Therefore, I thought either change the values on line 431 or remove the move endpoint. Do you know what I mean? Am I missing sth?

File: packages/shared/src/components/EditAltTextFlyout.tsx
Comment (celine-s) [reply packages/shared/src/components/EditAltTextFlyout.tsx:1]: goood point! Will add 👍

File: packages/animation-curve-block/src/AnimationCurveBlock.spec.ct.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ragi96) [reply packages/animation-curve-block/src/AnimationCurveBlock.spec.ct.tsx:424]: Then its all good

File: packages/shared/src/components/EditAltTextFlyout.spec.ct.tsx
Comment (ragi96) [inline packages/shared/src/components/EditAltTextFlyout.spec.ct.tsx:1]: Can we move them into a folder "EditAltTextFlyout" so it has the same structure as the other shared components

File: packages/shared/src/components/EditAltTextFlyout.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    FlyoutFooter,
+    FlyoutPlacement,
+    FormControl,
+    HelperPosition,
+    IconCheckMark16,
+    TextInput,
+} from '@frontify/fondue';
+import React from 'react';
+
+type EditAltTextFlyoutProps = {
```
Comment (ragi96) [inline packages/shared/src/components/EditAltTextFlyout.tsx:17]: Also move the types into a `types.ts` file within `EditAltTextFlyout` folder

**Review by ragi96 — COMMENTED** [review]
Just some structure adjustments, so it has the same structure as the other components within `shared`

File: packages/shared/src/components/EditAltTextFlyout.spec.ct.tsx
Comment (celine-s) [reply packages/shared/src/components/EditAltTextFlyout.spec.ct.tsx:1]: good point. Done 👍

File: packages/shared/src/components/EditAltTextFlyout.tsx (hunk 2 above)
Comment (celine-s) [reply packages/shared/src/components/EditAltTextFlyout.tsx]: 👍

**Review by ragi96 — APPROVED** [review]

**Pull request merged**